app/src/main/res/layout/activity_main.xml

```diff
@@ -7,12 +7,43 @@
     tools:context=".MainActivity">
 
     <TextView
```

Comment:
These views have to be removed in future commits since we are using a RecyclerView